GAE/src/org/waterforpeople/mapping/app/web/SurveyAssemblyServlet.java

```diff
@@ -264,106 +160,79 @@ protected void writeOkResponse(RestResponse resp) throws Exception {
     }
 
     /*
-     * One-pass assembly. May take too long for a single task if there are many questions.
-     * Upload to S3 might also be time-consuming.
+     * NEW! Assembly through Jackson classes.
+     *
      */
-    private boolean assembleSurveyOnePass(Long surveyId) {
-        log.info("Starting assembly of " + surveyId);
-        // Swap with proper UUID
+    private boolean assembleFormWithJackson(Long formId) {
+        log.info("Starting Jackson assembly of form " + formId);
```

Comment:
I would suggest its debugging log rather than info.

Reply:
But while it's so new I'd like to see this on a real instance, not just when running locally.